parm/atm/jcb-base.yaml.j2

```diff
@@ -11,7 +11,7 @@ app_path_observation_chronicle: "{{PARMgfs}}/gdas/jcb-gdas/observation_chronicle
 # ------------------------------------------------------------
 #final_increment_file: final_increment_gaussian
 final_increment_file: final_increment_cubed_sphere
-output_ensemble_increments_file: output_ensemble_increments_gaussian
```

Comment:
I think it might be better to add to the test YAML? I.e. add `output_ensemble_increments_file: output_ensemble_increments_cubed_sphere` to https://github.com/NOAA-EMC/GDASApp/blob/develop/test/atm/global-workflow/jcb-prototype_3dvar.yaml.j2. This will then override what is in jcb-base. Doing it that way isolates the changes from any external application. My sense is that these YAML files in the parm/atm should be governed by what we want to run in @RussTreadon-NOAA 's testing. As we evolve the application he is running we'll change these upper level YAMLs. But as we are developing and changing tests we limit to changing the YAML files in the test directory. This way it is sufficient for you to run the GDASapp tests and the global workflow tests but Russ does not need to run his testing until he's ready to make that change. Happy to discuss that though if it doesn't make sense.

Reply:
I guess the exception to this idea would be if the changes in global-workflow will inherently change the way Russ's tests would run. Then this change might be needed right away and you have it in the right place.

Reply:
@danholdaway The changes are not just intended for the jjob test, but also for how the JEDI LETFK application writes the increment in the Global Workflow. This will now mirror what is done for the variational analysis, whereby the the JEDI increment is written to the cubed sphere, and the increment converter read it and then writes out the FV3-readable increment in Gaussian format. So it would be just like this line 
 https://github.com/NOAA-EMC/GDASApp/blob/249e242e33a33feeb1c81bedd51198309f669de0/parm/atm/jcb-base.yaml.j2#L13 in jcb-base.yaml.j2 where we have "final_increment_cubed_sphere" instead of "final_increment_gaussian"

Reply:
The title of this PR is a little deceptive, since it doesn't just add a CTest.

Reply:
I see. Thanks for the clarification. Then keep as is and you can work with Russ to make sure everything is working properly.